api/v1beta1/cincinnati_types.go

```diff
@@ -50,31 +72,20 @@ const (
 	ConditionRegistryCACertFound conditionsv1.ConditionType = "RegistryCACertFound"
 )
 
-// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
-
-// Cincinnati is the Schema for a Cincinnati service.
+// +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
 // +kubebuilder:resource:path=cincinnatis,scope=Namespaced
+
+// Cincinnati is the Schema for the cincinnatis API
 type Cincinnati struct {
 	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	// metadata is standard object metadata.  More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
-	// +kubebuilder:validation:Required
-	metav1.ObjectMeta `json:"metadata"`
-
-	// spec is the desired state of the Cincinnati service.  The
```

Comment:
Dropping these partially reverts #55.  Can we keep the docstrings?

Reply:
Fixed in api/v1/updateservice_types.go